README.md

```diff
@@ -1,358 +1,48 @@
-};
-
-public function main() returns error? {
-    redis:Client conn =  check new (redisConfig);
-
-    io:println("Pinging Redis Server...");
-    //Ping Server
-    var result = conn->ping();
-    if (result is string) {
-        io:println(result);
-    } else {
-        io:println(result.message());
-    }
-
-    io:println("===Executing sample string oprerations===");
-    //Sample String Operations
-    io:println("Setting value of the key \"Project\" as \"Ballerina\"");
-    var stringSetresult = conn->set("Project", "Ballerina");
-
-    if (stringSetresult is string) {
-        io:println("Reply from the server: " + stringSetresult);
-    } else {
-        io:println("Error occurred while calling `set`");
-    }
-
-    io:println("\nQuerying the server for the value of the key \"Project\"");
-    var value = conn->get("Project");
-
-    if (value is string) {
-        io:println("Reply from the server: " + value);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `get`");
-    }
-
-    io:println("\n===Executing sample list oprerations===");
-    //Sample List Operations
-    io:println("Pushing 2 elements to NumberList");
-    var listPushresult = conn->lPush("NumberList", ["One", "Two"]);
+[![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
 
-    if (listPushresult is int) {
-        io:println(io:sprintf("Number of elements pushed: %s", listPushresult));
-    } else {
-        io:println("Error occurred while calling `lPush`");
-    }
+[Redis](https://redis.io/) is an open source, in-memory data structure store, used as a database, cache, and message broker. Redis provides data structures such as strings, hashes, lists, sets, sorted sets.
 
-    io:println("Poping an element from NumberList");
-    var lPopResult = conn->lPop("NumberList");
+This connector provides operations for connecting and interacting with Redis server over the network. 
+For more information about configuration and operations, go to the module. 
+- [`ballerinax/redis`](redis/Module.md)
 
-    if (lPopResult is string) {
-        io:println("Popped element: " + lPopResult);
-    } else if (result is ()) {
-        io:println("Key not found");
-    } else {
-        io:println("Error occurred while calling `lPop`");
-    }
-    io:println("===Executing sample set oprerations===");
-    //Sample Set Operations
-    io:println("Adding 3 elements to NumberSet");
-    var setAddResult = conn->sAdd("NumberSet", ["1", "2", "3"]);
+# Building from the Source
+## Setting Up the Prerequisites
 
-    if (setAddResult is int) {
-        io:println(io:sprintf("Number of elements added: : %s", setAddResult));
-    } else {
-        io:println("Error occurred while calling `sAdd`");
-    }
+1. Download and install Java SE Development Kit (JDK) version 11 (from one of the following locations).
```

Comment:
Suggestion:
Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).

Reply:
Will add

Reply:
As we have a common format across all the connectors. If we change this it will be inconsistent when compared with other connectors

Reply:
Changed